TROUBLESHOOTING.md

```diff
@@ -0,0 +1,10 @@
+# Cluster Logging Operator Troubleshooting
```

Comment:
Can we start a 'docs' folder

Reply:
we already have one, moving this to there